src/main/java/com/meilisearch/sdk/KeysHandler.java

```diff
@@ -45,6 +46,18 @@ Results<Key> getKeys() throws MeilisearchException {
         return result;
     }
 
+    /**
+     * Retrieves keys from the client
```

Comment:
```suggestion
     * Get API keys
```

Reply:
"Retrieves" is use in the definitions of all get method for documents, indexes and tasks so it should be changes in all of it. I can replace it by:
`Retrieves API keys from the client`
?

Reply:
I will open an issue about replacing all the retrieves by get